transports/rabbitmq/delayed-delivery.md

```diff
@@ -50,10 +56,43 @@ end
+|-----------------------|------------------------|-------------------------------------------------------------|
+| nsb.delay-level-27    | nsb.delay-level-26     | `1.#`                                                       |
+| nsb.delay-level-26    | nsb.delay-level-25     | `*.1.#`                                                     |
+| nsb.delay-level-25    | nsb.delay-level-24     | `*.*.1.#`                                                   |
+| ...                   | ...                    | ...                                                         |
+| nsb.delay-level-03    | nsb.delay-level-02     | `*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.1.#`       |
+| nsb.delay-level-02    | nsb.delay-level-01     | `*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.1.#`     |
+| nsb.delay-level-01    | nsb.delay-level-00     | `*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.1.#`   |
+| nsb.delay-level-00    | nsb.delay-delivery     | `*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.*.1.#` |
+
+With this system of exchanges and queues, the message is first delivered to the first applicable queue based on the binary value. In the 10-second example (binary `1010`) the message would first be delivered to the `nsb.delay-level-03` exchange, where the matching queue's binding will only match if a `1` exists in the 4th-to-last digit. In that case the queue will "accept" the message into the `nsb.delay-level-03` queue and wait for something to process it up until the TTL (2^3 seconds) before forwarding to the `nsb.delay-level-02` exchange.
+
+When the bindings for exchanges and queues don't match up, i.e. when the bit value is zero, wildcard bindings with 0 values instead of 1 ensure that messages that shouldn't be delayed for that delay level are immediately forwarded to the next delay level. In other words, for each exchange, the bit value in a message's routing key will either match `1` and be forwarded to the queue to be delayed by the TTL, or will match `0` and be forwarded to the next delay level.
+
+This process continues all the way through `nsb.delay-level-00`, where the x-dead-letter-exchange value (for matching routing keys) or the exchange-to-exchange binding (for non-matching routing keys) forward the message to the `nsb.delay-delivery` exchange.
+
 
 ### Delivery
 
-The final delay-level exchange is bound to the delivery exchange instead of another delay level. Every endpoint that can receive a delayed message will have a binding to this exchange with a routing key corresponding to the endpoint's name, so the message will be delivered to the endpoint's queue.
+It is at the final step, in the `nsb.delay-delivery` exchange, where the real message destination is evaluated to determine which queue (based on endpoint name) the message should be delivered to.
+
+Each endpoint creates a binding to the `nsb.delay-delivery` exchange, similar to the following:
```

Comment:
Getting into this level of detail, you run into the fact that a portion of the delayed delivery implementation is topology-specific. The binding created to the `nsb.delay-delivery` exchange is controlled by the routing topology. For the conventional topology, this is an exchange-to-exchange binding between `nsb.delay-delivery` and the endpoint's exchange, but for the direct routing topology, this binding is made between `nsb.delay-delivery` and the actual queue. And of course, a custom routing topology is free to implement this however it needs to.

The routing topology has to implement the following method: 
`public void BindToDelayInfrastructure(IModel channel, string address, string deliveryExchange, string routingKey)`

so it does get the delivery exchange and routing key value, but is free to use those in whatever way is needed.

Anyway, that's a long of way of saying that "destination queue" in this section isn't entirely correct, but I'm not sure how correct you actually want to be here.